server/src/database/models/Product_order.ts

```diff
@@ -0,0 +1,16 @@
+import { DataTypes } from "sequelize";
+import sequelize from "../config/connection";
+
+const Product_Order = sequelize.define('product_order',{
```

Comment:
use camelcase for variable names like `ProductOrder`

and you should fix all your eslint errors